index.html

```diff
@@ -93,25 +91,18 @@
     <section>
       <h2>Initializing fetch settings</h2>
 
-      <p>The <code><dfn>crossorigin</dfn></code> <a href="https://html.spec.whatwg.org/multipage/infrastructure.html#cors-settings-attributes">CORS setting attribute</a> is an OPTIONAL attribute that indicates the CORS policy of the specified resource.</p>
-
-      <p>The <code><dfn>as</dfn></code> attribute on a <a>preload</a> relationship specifies the <a href="https://fetch.spec.whatwg.org/#concept-request-context">request context</a> used to initialize appropriate fetch settings - e.g. request priority, HTTP headers, etc.</p>
-
       <pre class="example highlight html">
-&lt;link rel="preload" href="/assets/font.woff" as="font"&gt;
-&lt;link rel="preload" href="/assets/logo.webp" as="image"&gt;
-&lt;link rel="preload" href="//example.com/widget" as="iframe"&gt;
+&lt;link rel="preload" href="/asset/font.woff" type="application/font-woff"&gt;
+&lt;link rel="preload" href="/asset/logo.webp" type="image/webp"&gt;
+&lt;link rel="preload" href="//example.com/other.jpeg" type="image/jpeg" crossorigin&gt;
+&lt;link rel="preload" href="//example.com/widget" type="text/html"&gt;
       </pre>
 
       <ul>
-        <li>The <code><a>as</a></code> attribute is a REQUIRED attribute for a <a>preload</a> relationship. The specified <code><a>as</a></code> value MUST be a valid <a href="http://fetch.spec.whatwg.org/#concept-request-context">request context</a> as defined in [[FETCH]]. If the <code><a>as</a></code> attribute is omitted, or the specified value does not contain a valid <a href="http://fetch.spec.whatwg.org/#concept-request-context">request context</a>, then the user agent SHOULD output a developer-friendly warning and ignore the <a>preload</a> relationship.</li>
+        <li>The <code><dfn>crossorigin</dfn></code> <a href="https://html.spec.whatwg.org/multipage/infrastructure.html#cors-settings-attributes">CORS setting attribute</a> is an OPTIONAL attribute that indicates the CORS policy of the specified resource.</li>
 
-        <li>Request defaults set by the user agent via <code><a>as</a></code> attribute MUST match the default settings set by the user agent when processing a resource with the same context. This behavior is necessary to guarantee correct prioritization and request matching - see <a href="#matching-responses-with-requests"></a>.</li>
+        <li>The <code><dfn><a href="http://www.w3.org/TR/html5/links.html#attr-hyperlink-type">type</a></dfn></code> attribute is an OPTIONAL attribute whose value MUST be a <a href="http://www.w3.org/TR/html5/infrastructure.html#valid-mime-type">valid MIME type</a> and MAY be used by the user agent to initialize appropriate <a href="https://fetch.spec.whatwg.org/#fetching">fetch settings</a> - e.g. HTTP request headers, request and transport priorities, etc.</li>
```

Comment:
I think @marcoscaceres' was objecting to having the mix of MAY and MUST (otherwise, they are just authoring guidelines which are meh, IMO). The excepted values for the attribute are defined by the element itself (as well as what values the attribute expects). I would expect an algorithm somewhere to define how the UA behaves when an invalid value is given, as well as how processed attribute values are reflected in the IDL attributes of an element.